orca/templates/antivirus/Dockerfile

```diff
@@ -1,2 +1,2 @@
 #FROM clamav/clamav:0.105
```

Comment:
The tag of the image did not change? Please confirm

Reply:
Oops, thanks for your eyes, it's been fixed now.